providers/summary/scancode.js

```diff
@@ -47,9 +47,17 @@ class ScanCodeSummarizer {
     if (releaseDate) result.described = { releaseDate: extractDate(releaseDate.trim()) }
   }
 
-  _readDeclaredLicense(harvested) {
-    const declared = get(harvested, 'content.summary.packages[0].declared_license')
-    return SPDX.normalize(declared)
+  _readDeclaredLicense(scancodeVersion, harvested) {
+    switch (scancodeVersion) {
+      case '2.2.1':
+      case '2.9.2':
+      case '2.9.8':
+      case '3.0.0':
+      case '3.0.2':
+        return SPDX.normalize(get(harvested, 'content.summary.packages[0].declared_license'))
+      case '30.1.0':
+        return SPDX.normalize(get(harvested, 'content.summary.packages[0].declared_license[0]'))
+    }
```

Comment:
should there be a default/fallthrough clause to throw or err out if a different version just happens to make it to this codepath?

Reply:
that's a good note - taking a look